web/templates/settings/custom_user_profile_field.hbs

```diff
@@ -7,9 +7,9 @@
     <div class="settings-profile-user-field-hint">{{ field.hint }}</div>
     <div class="settings-profile-user-field {{#if is_empty_required_field}}empty-required-field{{/if}}">
         {{#if is_long_text_field}}
-        <textarea maxlength="500" class="custom_user_field_value settings_textarea" name="{{ field.id }}">{{ field_value.value }}</textarea>
+        <textarea id="{{field.name}}" maxlength="500" class="custom_user_field_value settings_textarea" name="{{ field.id }}">{{ field_value.value }}</textarea>
```

Comment:
I am not sure this is safe; are these field names guaranteed to be unique in any way? IDs must be globally unique across the app, and as written, I'm pretty sure a field name of the form.

Probably you want to use something like `id_custom_profile_field_input_{id}` for this value, not `field.name`.

Reply:
From what I know, the `for` value from the label uses `{{ field.name }}` so it could be assumed that it was used as id before this. But yeah I've changed it to use `id_custom_profile_field_input_{id}` so it's guaranteed to be unique.